libraries/Microsoft.Bot.Builder/Teams/TeamsActivityHandler.cs

```diff
@@ -994,6 +1001,69 @@ protected virtual Task OnTeamsMessageSoftDeleteAsync(ITurnContext<IMessageDelete
             return Task.CompletedTask;
         }
 
+        /// <summary>
+        /// Invoked when a feedback loop activity is received.
```

Comment:
It would be more accurate to say "Invoked when a user submits a default or custom feedback loop form".